consul/install.go

```diff
@@ -0,0 +1,78 @@
+package consul
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/layer5io/gokit/errors"
+
+	"github.com/layer5io/meshery-consul/internal/config"
+
+	"github.com/layer5io/meshery-adapter-library/adapter"
+)
+
+type MeshInstance struct{}
+
+func (m *MeshInstance) applyUsingManifest(request adapter.OperationRequest, operation *adapter.Operation, h *Handler) error {
```

Comment:
Any particular reason to pass operation as a pointer?

Reply:
No. I change it to pass by value, the value shouldn't be changed.